googlesearch/__init__.py

```diff
@@ -165,40 +166,33 @@ def get_page(url, user_agent=None, verify_ssl=True):
     :raises urllib2.URLError: An exception is raised on error.
     :raises urllib2.HTTPError: An exception is raised on error.
     """
-    if user_agent is None:
-        user_agent = USER_AGENT
+    user_agent = USER_AGENT if user_agent is None else user_agent
     request = Request(url)
     request.add_header('User-Agent', user_agent)
     cookie_jar.add_cookie_header(request)
-    if verify_ssl:
-        response = urlopen(request)
-    else:
-        context = ssl._create_unverified_context()
-        response = urlopen(request, context=context)
+    :
+    response = urlopen(request) if verify_ssl urlopen(request, context=ssl._create_unverified_context())
```

Comment:
what is this?
did you use an automatic code generator to rewrite the code?
this is completely wrong.

Reply:
I'm not sure what I'm looking at either here honestly, that's not right at all.